tsdb/mapper.go

```diff
@@ -0,0 +1,664 @@
+package tsdb
+
+import (
+	"encoding/binary"
+	"fmt"
+	"time"
+
+	"github.com/boltdb/bolt"
+	"github.com/influxdb/influxdb/influxql"
+)
+
+// RawMapper is for retrieving data, for a query, from a given shard. It implements the
```

Comment:
The `ShardMapper` interface doesn't actually exist, and this comment needs correction.